Build ARM32v7 error: call of overloaded ‘abs(__gnu_cxx::__alloc_traits<std::allocator<float> >::value_type)’ is ambiguous

**Describe the bug**
I met an error message while cross compiling for ARM with Docker.
I follow the instructions in [this page](https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/README.md#arm-32v7)

At step 4. Run docker build,
```
sudo docker build -t onnxruntime-arm32v7 -f Dockerfile.arm32v7 .
```
The error message:
```
[100%] Building CXX object CMakeFiles/onnxruntime_global_thread_pools_test.dir/code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc.o
In file included from /code/onnxruntime/cmake/external/googletest/googletest/include/gtest/gtest.h:62:0,
                 from /code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc:14:
/code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc: In instantiation of ‘void RunSession(OrtAllocator&, Ort::Session&, std::vector<Input>&, const char*, const std::vector<long long int>&, const std::vector<T>&, Ort::Value*) [with OutT = float; OrtAllocator = OrtAllocator]’:
/code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc:108:19:   required from ‘void TestInference(Ort::Session&, std::vector<Input>&, const char*, const std::vector<long long int>&, const std::vector<OutT>&) [with T = const char*; OutT = float]’
/code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc:157:109:   required from here
/code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc:61:20: error: call of overloaded ‘abs(__gnu_cxx::__alloc_traits<std::allocator<float> >::value_type)’ is ambiguous
     ASSERT_TRUE(abs(values_y[i] - f[i]) < 1e-6);
                 ~~~^~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/6/cstdlib:75:0,
                 from /usr/include/c++/6/ext/string_conversions.h:41,
                 from /usr/include/c++/6/bits/basic_string.h:5417,
                 from /usr/include/c++/6/string:52,
                 from /usr/include/c++/6/stdexcept:39,
                 from /usr/include/c++/6/array:39,
                 from /usr/include/c++/6/tuple:39,
                 from /usr/include/c++/6/functional:55,
                 from /usr/include/c++/6/memory:79,
                 from /code/onnxruntime/include/onnxruntime/core/common/make_unique.h:28,
                 from /code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc:4:
/usr/include/stdlib.h:735:12: note: candidate: int abs(int)
 extern int abs (int __x) __THROW __attribute__ ((__const__)) __wur;
            ^~~
In file included from /usr/include/c++/6/ext/string_conversions.h:41:0,
                 from /usr/include/c++/6/bits/basic_string.h:5417,
                 from /usr/include/c++/6/string:52,
                 from /usr/include/c++/6/stdexcept:39,
                 from /usr/include/c++/6/array:39,
                 from /usr/include/c++/6/tuple:39,
                 from /usr/include/c++/6/functional:55,
                 from /usr/include/c++/6/memory:79,
                 from /code/onnxruntime/include/onnxruntime/core/common/make_unique.h:28,
                 from /code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc:4:
/usr/include/c++/6/cstdlib:180:3: note: candidate: long long int std::abs(long long int)
   abs(long long __x) { return __builtin_llabs (__x); }
   ^~~
/usr/include/c++/6/cstdlib:172:3: note: candidate: long int std::abs(long int)
   abs(long __i) { return __builtin_labs(__i); }
   ^~~
CMakeFiles/onnxruntime_global_thread_pools_test.dir/build.make:75: recipe for target 'CMakeFiles/onnxruntime_global_thread_pools_test.dir/code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc.o' failed
make[2]: *** [CMakeFiles/onnxruntime_global_thread_pools_test.dir/code/onnxruntime/onnxruntime/test/global_thread_pools/test_inference.cc.o] Error 1
CMakeFiles/Makefile2:807: recipe for target 'CMakeFiles/onnxruntime_global_thread_pools_test.dir/all' failed
make[1]: *** [CMakeFiles/onnxruntime_global_thread_pools_test.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
Traceback (most recent call last):
  File "/code/onnxruntime/tools/ci_build/build.py", line 1530, in <module>
    sys.exit(main())
  File "/code/onnxruntime/tools/ci_build/build.py", line 1432, in main
    build_targets(args, cmake_path, build_dir, configs, args.parallel)
  File "/code/onnxruntime/tools/ci_build/build.py", line 745, in build_targets
    run_subprocess(cmd_args, env=env)
  File "/code/onnxruntime/tools/ci_build/build.py", line 352, in run_subprocess
    env=my_env, shell=shell)
  File "/usr/lib/python3.5/subprocess.py", line 398, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/usr/local/bin/cmake', '--build', '/code/onnxruntime/build/Linux/MinSizeRel', '--config', 'MinSizeRel']' returned non-zero exit status 2
The command '/bin/sh -c ./build.sh ${BUILDARGS} --build_shared_lib' returned a non-zero code: 1
```

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04 LTS
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: master branch from git
- Python version: 3.6.9
- GCC/Compiler version (if compiling from source): 7.5.0

--- a/onnxruntime/test/global_thread_pools/test_inference.cc
+++ b/onnxruntime/test/global_thread_pools/test_inference.cc
@@ -58,7 +58,7 @@ static void RunSession(OrtAllocator& allocator, Ort::Session& session_object,
 
   OutT* f = output_tensor->GetTensorMutableData<OutT>();
   for (size_t i = 0; i != static_cast<size_t>(5); ++i) {
-    ASSERT_TRUE(abs(values_y[i] - f[i]) < 1e-6);
+    ASSERT_TRUE(std::abs(values_y[i] - f[i]) < 1e-6);
   }
 }
 

